composer.json

```diff
@@ -20,9 +20,10 @@
         }
     },
     "require": {
-        "php": "^7.0"
+        "php": "^7.4|^8.0",
```

Comment:
Is it really necessary to require 7.4?

Reply:
No, but due to the behaviour change in #7, I was planning to release 2.0 and why not require a recent PHP version for that?

Reply:
So, do you mind releasing 2.0 or should I remove the code the requires 7.4?